SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -1667,8 +1667,37 @@ void USpatialNetDriver::ProcessRemoteFunction(
 	}
 }
 
+void USpatialNetDriver::PollPendingLoads()
+{
+	if (PackageMap)
+	{
+		for (USpatialPackageMapClient::PendingRefSet::TIterator IterPending = PackageMap->PendingReferences.CreateIterator(); IterPending; ++IterPending)
+		{
+			if (!PackageMap->IsGUIDPending(*IterPending))
+			{
+				FUnrealObjectRef ObjectReference = PackageMap->GetUnrealObjectRefFromNetGUID(*IterPending);
```

Comment:
That's a reasonable suggestion, but this would be a reference to an entry in a TMap which, if I'm not mistaken, uses a TArray backing store. It would work 99% of the time until someone keeps a ref, modifies the map, causes a realloc and we have a random crash. While we could be super careful around these methods, I would err on the safe side here.

Reply:
Your issue is with dangling references?

Reply:
Yes.

Reply:
So the concern is that a reference can be made invalid after the map has be deleted or possibly after it has been modified?

Reply:
Yes, usually what eventually happens when you take references to something in a std::vector for instance. No issue with a std::map since each map entry is individually allocated.